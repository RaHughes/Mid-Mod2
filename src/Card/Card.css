.card {
  height: auto;
  width: 250px;
  background-color: white;
  display: flex;
  flex-direction: column;
  border: 4px solid teal;
  margin: 5%;
  padding: 2%;
}

h2 {
  color: teal;
}

p {
  color: teal;
}

.submit {
  background-color: teal;
  color: white;
  width: auto;
  height: 30px;
}