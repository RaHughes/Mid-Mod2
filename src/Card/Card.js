import React from 'react';
import './Card.css';

const Card = ({date, name, number, time, deleteData, id}) => {
  return (
    <article className='card'>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time} PM</p>
      <p>Number of Guests:{number}</p>
      <button className='submit' onClick={() => deleteData(id)}>Cancel</button>
    </article>
  )
}

export default Card;