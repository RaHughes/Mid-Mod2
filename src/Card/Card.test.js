import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Card', () => {
  it('Should match snapshot', () => {
    const mockData = {
      key: 1,
      date: '',
      id: 2,
      name: '',
      number: 2,
      time: ''
    }
    const wrapper = shallow(<Card key={mockData.id} date={mockData.date} id={mockData.id} name={mockData.name} number={mockData.number} time={mockData.time}/>)
    expect(wrapper).toMatchSnapshot()
  });
})