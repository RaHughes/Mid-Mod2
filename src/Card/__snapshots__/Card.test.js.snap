// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should match snapshot 1`] = `
<article
  className="card"
>
  <p />
  <p />
  <p />
  <p>
    Number of Guests:
    2
  </p>
  <button>
    Cancel
  </button>
</article>
`;
